docs/modules/ROOT/pages/how-tos/configuring/prefetching-dependencies.adoc

```diff
@@ -1,7 +1,251 @@
 = Prefetching package manager dependencies for hermetic builds
 
+//NOTE: We may want to rephrase this to word for more accurate SBOM information
+//NOTE: Should this be a subtopic for hermetic or a sibling to it?
```

Comment:
I think a sibling. There may be use cases where prefetching is beneficial even without hermetic builds.